[DONE] Create account

Tonirel opened the pull request:
We just need to add what happens when you press the buttons "Create account" and "Cancel" (there are "todo" comments inside the code) which will happen after the other functionalities are finished + change the white background which will happen after the UI design is finished.

Commit:
Added create account

schiaudoriana committed:
create account done

andreeasechelaru commented:
> We just need to add what happens when you press the buttons "Create account" and "Cancel" (there are "todo" comments inside the code) which will happen after the other functionalities are finished + change the white background which will happen after the UI design is finished.

schiaudoriana committed:
removed create account from app component

schiaudoriana committed:
Merge remote-tracking branch 'origin/main' into create-account

# Conflicts:
#	package-lock.json
#	package.json
#	src/app/add-expense/add-expense.component.html
#	src/app/add-expense/add-expense.component.spec.ts
#	src/app/add-expense/add-expense.component.ts
#	src/app/app-routing.module.ts
#	src/app/app.component.html
#	src/app/app.component.spec.ts
#	src/app/app.module.ts
#	src/environments/environment.prod.ts
#	src/environments/environment.ts
#	src/index.html
#	src/styles.css

schiaudoriana committed:
updated project + added some styles

Commit:
Interface attempt (failed)

Tried to make the interface for the "create account" functionality but failed to do so. Will ask for help/research and try again another time. (kept old html code as comments just in case)

Commit:
Almost done!

Looks way better than before, I'll ask for help in the next week to finalize the interface (I can't figure out how to space things out properly)

Commit:
Removed extra stuff from CSS folder

schiaudoriana committed:
create account page styling done (hopefully)

Commit:
removed empty ruleset from css

andreeasechelaru commented on src/app/create-account/create-account.component.ts:
```diff
@@ -0,0 +1,77 @@
+
+  onCancel() {
+    //todo: navigate to the previous page
+    console.log("on cancel")
```
in ce cazuri se apeleaza functia asta?

Tonirel replied:
Ai dreptate, pentru ca nu avem buton de cancel, functia nu se apeleaza niciodata. O comentez, in caz ca avem nevoie de ea pe viitor

Commit:
fixed build errors and fixed old styling issues

adipopbv reviewed (approved):
Good shit

CristianRotar commented on src/app/create-account/create-account.component.html:
```diff
@@ -178,14 +176,59 @@ <h1> Register </h1>
   </div>
 </div>
+-->
+<!--Attempt 3 (I promise it's the final!)-->
```
Attempt 3 looks good, so you can delete the first 2 attempts

Tonirel replied:
I will!

Commit:
Removed old html code and commented "onCancel" function
